COM-599 Update testing report template

lauravignoli opened the pull request.

lauravignoli committed:
COM-599 Update testing report template

tmvumbi2 commented on openmrs/apps/reports/TESTING_REPORT/sql/indicator1a_not_available.sql:
```diff
@@ -0,0 +1,53 @@
+    0 AS '1 - 4 years M Pos',
+    0 AS '1 - 4 years F Pos',   
+    0 AS '5 - 9 years M Pos',
+    0 AS '1 - 4 years F Pos',
```
@lauravignoli did u mean `'5 - 9 years F Pos',` here?

tmvumbi2 commented on openmrs/apps/reports/TESTING_REPORT/sql/indicator1a_not_available.sql:
```diff
@@ -0,0 +1,53 @@
+    0 AS '1 - 4 years M Neg',
+    0 AS '1 - 4 years F Neg',   
+    0 AS '5 - 9 years M Neg',
+    0 AS '1 - 4 years F Neg',
```
@lauravignoli did you mean `'5 - 9 years F Neg'` here?

tmvumbi2 committed:
COM-599 Fix copy-paste errors
- there were copy/paste errors on line 7 and 33 of the indicator1 SQL files (line 7 and 15 for indicator1d_not_available.sql)
- indicator5e_not_available.sql should not have male columns (similar to Indicator 5d)
